pkgs/development/python-modules/manimpango/default.nix

```diff
@@ -0,0 +1,85 @@
+{ lib
+, buildPythonPackage
+, pkg-config
+, fetchPypi
+, cairo
+, pango
+, pygobject3
+, glib
+, libintl
+, harfbuzz
+, cython
+, setuptools
+, pytest
+, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "manimpango";
+  version = "0.2.6";
+
+  # doc/languages-frameworks/python.section.md: [..] Attribute names in
+  # `python-packages.nix` as well as `pname`s should match the library's name
+  # on PyPI, but be normalized according to [PEP
+  # 0503](https://www.python.org/dev/peps/pep-0503/#normalized-names). This
+  # means that characters should be converted to lowercase [..]
+  src = fetchPypi {
+    pname = "ManimPango";
+    version = "0.2.6";
+    sha256 = "64028b62b151bc80b047cc1757b27943498416dc4a85f073892a524b4d90ab41";
+  };
+
+  # inputs only needed during build
+  nativeBuildInputs = [
+    pkg-config
+    setuptools
+    cython
+  ];
+
+  # non-python deps
```

Comment:
```suggestion
```

Reply:
is this wrong?

Reply:
Those comments don't add any value to the package. We are in the pythonPackage part of nixpkgs and every package here is either a pythonPackage or a build/runtime dependency.